examples/wordcount/example.py

```diff
@@ -15,25 +15,29 @@
 
 log = logging.getLogger(__name__)
 
```

Comment:
Thought this would be a nice example to show how punctuate can debounce a high frequency input stream and publish updated counts only every few seconds at most.